firefox-ios/firefox-ios-tests/Tests/XCUITests/ScreenGraphTest.swift

```diff
@@ -136,7 +136,7 @@ private func createTestGraph(for test: XCTestCase, with app: XCUIApplication) ->
     map.addScreenState(URLBarOpen) { screenState in
         screenState.gesture(forAction: TestActions.LoadURLByTyping, TestActions.LoadURL) { userState in
             let urlString = userState.url ?? defaultURL
-            app.textFields["address"].typeText("\(urlString)\r")
+            urlBarAddress.typeText("\(urlString)\r")
```

Comment:
@dragosb01 Sorry. The l10n-screenshots tests are not compiling at the moment because `urlBarAddress` is not defined here. Could you please fix this?

```
[23:14:53]: ▸ ❌  /Users/cso/workspace/firefox-ios/firefox-ios/firefox-ios-tests/Tests/XCUITests/ScreenGraphTest.swift:139:13: cannot find 'urlBarAddress' in scope
[23:14:53]: ▸             urlBarAddress.typeText("\(urlString)\r")
[23:14:53]: ▸ ~~~~~~~~~~~~~~~~^
```

Reply:
You can run the L10nBuild locally by `./firefox-ios/l10n-screenshots.sh en-US` provided you have `fastlane` installed.
https://github.com/mozilla-mobile/firefox-ios/blob/main/bitrise.yml#L838

Reply:
@dragosb01 reason is that they don't inherit from BaseTestCase.... `ScreenGraphTest: XCTestCas` so the element defined there is not available, you would need to define it here or leave it as it was